index.js

```diff
@@ -9,11 +9,12 @@ module.exports = {
     'indent': [2, 2, { SwitchCase: 1 }],
     'key-spacing': [2, { 'beforeColon': false, 'afterColon': true }],
     'linebreak-style': [2, 'unix'],
-    'max-depth': [2, 3],
+    "max-depth": ["error", 5],
```

Comment:
Well it is better to have it here, and accept we need more depth on some code sections, than random linting exceptions blocks around the code when we don't like the rules.